README.md

```diff
@@ -1,2 +1,93 @@
+
+#### Returning
+
+Issues affixed with the `Returning` label are placed in the `Upcoming` status on the [Project Board](https://github.com/orgs/cncf/projects/14) for an updated discussion at the next Sandbox Application meeting.
+
+## How applications are reviewed
+
+### Frequency
+
+The TOC reviews sandbox applications approximately **every two months** as of June 2020 in a non-public meeting; referred to as "session" hereafter. 
+
+### Quantity
+
+The TOC attempts to work through approximately 7-10 applications per session. How many are actually reviewed in a session varies greatly between each session as every project is unique and different areas of attention may be elevated for more in depth discussion depending on the nature and function of the project.
+
+Each TOC member prepares for these sessions by performing an independent review of those projects scheduled for discussion at the next meeting. They'll keep their notes on hand for when the project is up for discussion.
+
+### Review order
+
+Sandbox applications are traditionally reviewed in a First In, First Out (FIFO) ordering as they appear on the **🏗 Upcoming** Column of the [Sandbox Application Board](https://github.com/orgs/cncf/projects/14/views/1) by their corresponding issue number. The only exception is where projects are `Returning` for review. Projects that are `Returning` are prioritized for re-review as they had previously been subject to a review and have completed their outstanding asks.
+
+Applications are moved from `New` to `Upcoming` approximately two weeks prior to the next scheduled session by the TOC or support staff.
+
+### Discussions and decision
+
+During the course of the session, the TOC will actively discuss the project based on the information provided in the issue as well as other observations the TOC has about the project. Those observations vary greatly and may be subject to each TOC member's area of domain expertise, personal experience, community feedback, or other factors. During the course of active discussion, the TOC may identify areas the project needs to work on, complete, or allow to happen that could inhibit its acceptance into the CNCF. These areas will be captured succinctly and affixed the corresponding label ([see Label statuses defined](#label-statuses-defined) for more information). The issue will also receive a comment that details the additional areas that correspond with the label.
+
+#### Examples
+
+All project and organization names in the examples are fictional and are not intended to reflect any existing project past or present.
+
+##### Scuppers
```

Comment:
"Skupper" is an actual cloud native space project; suggest using a different name to avoid reader confusion.  Suggestion: "Baggywrinkle" (an actual nautical term, really) (alternate: Goosewing)

Reply:
Further suggestion: Yawl.
"Yawl - a two-masted boat with the mizzen stepped aft of the rudder stock / post."